@xen-orchestra/web-core/lib/components/card/VtsCardRowKeyValue.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="vts-card-row-key-value typo p3-regular">
+    <div class="key">
+      <slot name="key" />
+    </div>
+    <div v-tooltip class="content text-ellipsis">
```

Comment:
I made a mistake in my first suggestion. For consistency, it would be better to have the class match the slot name:
```suggestion
    <div v-tooltip class="value text-ellipsis">
```